scripts/base_network.py

```diff
@@ -768,6 +1050,16 @@ def base_network(
     n.import_components_from_dataframe(links, "Link")
     n.import_components_from_dataframe(converters, "Link")
 
+    for project, include in line_projects.include_projects.items():
+        if include:
```

Comment:
I think it could make sense to rewrite this so it doesn't need to iterate over NEP and TYNDP. Maybe just concatenate identically named files in each considered folder at the beginning.

Reply:
Not quite sure that I understand what you mean here.